app/javascript/components/pages/DrinkShow.js

```diff
@@ -32,10 +37,14 @@ class DrinkShow extends Component {
              src= {drink.drinkThumb}
              alt = "drink img"
              />
-             <Button onClick={this.handleSubmit}>Add To Favorites</Button>
+             {
+               logged_in ? logged_in &&   <Button onClick={this.handleSubmit}>Add To Favorites</Button>
+               :null
+             }
```

Comment:
looks like it can be simplified so that we don't have to check on `logged_in` twice
```suggestion
             {
               logged_in &&   <Button onClick={this.handleSubmit}>Add To Favorites</Button>
             }
```